### PR TITLE
Update validation for OME namespace

### PR DESCRIPTION
**Pull request by konantian**

### CODE REVIEW

File: bids-validator/validators/microscopy/ometiff.js [hunk 1]
```diff
@@ -77,10 +84,15 @@ const checkOptionalFields = (omePath, omeData, jsonData) => {
   }
 
   if (
-    omeData['OME']['Instrument'] &&
-    omeData['OME']['Instrument'][0]['Objective']
+    omeData[`${namespace}OME`][`${namespace}Instrument`] &&
+    omeData[`${namespace}OME`][`${namespace}Instrument`][0][
+      `${namespace}Objective`
+    ]
```
Comment (mariehbourget) [inline bids-validator/validators/microscopy/ometiff.js]:
```suggestion
    omeData[`${namespace}OME`][`${namespace}Instrument`][0][`${namespace}Objective`]
```
The formatting on 3 lines seems odd to me, can it be on 1 line?

File: bids-validator/validators/microscopy/ometiff.js [hunk 2]
```diff
@@ -77,10 +84,15 @@ const checkOptionalFields = (omePath, omeData, jsonData) => {
...
   ) {
-    let objective = omeData['OME']['Instrument'][0]['Objective'][0]['$']
+    let objective =
+      omeData[`${namespace}OME`][`${namespace}Instrument`][0][
+        `${namespace}Objective`
+      ][0]['$']
```
Comment (mariehbourget) [inline bids-validator/validators/microscopy/ometiff.js]: Same as above

File: bids-validator/validators/microscopy/ometiff.js [hunk 3]
```diff
@@ -103,22 +115,34 @@ const checkOptionalFields = (omePath, omeData, jsonData) => {
...
-const checkPixelSize = (omeData, jsonData) => {
+const checkPixelSize = (omeData, namespace, jsonData) => {
   let issues = []
   let validUnits = ['um', 'µm', 'nm', 'mm']
 
   const PhysicalSizeX =
-    omeData['OME']['Image'][0]['Pixels'][0]['$']['PhysicalSizeX']
+    omeData[`${namespace}OME`][`${namespace}Image`][0][`${namespace}Pixels`][0][
+      '$'
+    ]['PhysicalSizeX']
```
Comment (mariehbourget) [inline bids-validator/validators/microscopy/ometiff.js]: Same as above, and same for the 5 other occurrences below.

**Review by mariehbourget — APPROVED** [review]
Tested with different datasets (`OME` and `ome:OME`), it works as expected.
Thanks @konantian!

I left some minor formatting suggestions.

File: bids-validator/validators/microscopy/ometiff.js (hunk 1 above)
Comment (konantian) [reply bids-validator/validators/microscopy/ometiff.js]: This format is suggested by eslint, it looks odd to me as well. I will update them then.

**Pull request merged**